app/services/data/insert-offender-manager.js

```diff
@@ -0,0 +1,19 @@
+const config = require('../../../knexfile').development
+const knex = require('knex')(config)
+const offenderManagerTable = `${config.DB_STG_SCHEMA}.offender_manager`
+
+module.exports = function (offenderManager) {
+  var offenderManagerId
+
+  knex.select().from(offenderManagerTable)
+    .where('om_key', offenderManager.omKey)
+    .first()
```

Comment:
yes. Though there shouldn't be more than one offender manager with the same omKey